src/applications/claims-status/tests/components/claim-files-tab/RemoveFileModal.unit.spec.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import RemoveFileModal from '../../../components/claim-files-tab/RemoveFileModal';
+
+describe('<RemoveFileModal>', () => {
+  it('should render component', () => {
+    const removeFileName = 'test.pdf';
+    const showRemoveFileModal = true;
+    const onRemoveFile = sinon.spy();
+    const onCloseModal = sinon.spy();
+
+    const { container } = render(
+      <RemoveFileModal
+        removeFileName={removeFileName}
+        showRemoveFileModal={showRemoveFileModal}
+        closeModal={onCloseModal}
+        removeFile={onRemoveFile}
+      />,
+    );
+
+    expect($('#remove-file', container)).to.exist;
+    expect($('va-modal', container).getAttribute('primarybuttontext')).to.equal(
+      'Yes, remove this',
+    );
+    expect(
+      $('va-modal', container).getAttribute('secondarybuttontext'),
+    ).to.equal('No, keep this');
+  });
+
+  it('shows the filename to be removed', () => {
+    const removeFileName = 'test.pdf';
+    const showRemoveFileModal = true;
+    const onRemoveFile = sinon.spy();
```

Comment:
do you need the spys here...doesnt look like there are tests around them so id just remove them.